packages/test/test-end-to-end-tests/src/test/summarization/summarizeNode.spec.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+import { describeCompat } from "@fluid-private/test-version-utils";
+import { LoaderHeader } from "@fluidframework/container-definitions/internal";
+import { IFluidHandle, IRequestHeader } from "@fluidframework/core-interfaces";
+import {
+	ITestFluidObject,
+	ITestObjectProvider,
+	createTestConfigProvider,
+	createSummarizerFromFactory,
+	summarizeNow,
+} from "@fluidframework/test-utils/internal";
+
+/**
+ * The test is skipped as we have an existing bug in our code where if the use of short ids for data stores and dds is enabled, the summaries will start failing.
+ * Without short ids, the code works because the handle for a tree node is exactly able to point to a path of node in the summary tree, because the ids do not have any special characters which
+ * could change with the encodeURIComponent(). Once shortids feature is enabled in container runtime, the ids can have special characters (like '[' ) which can result in a handle containing %5b -
+ * this will be problematic because when uploading a summary with such handles, the server will try to look for a path with %5b but will not find it, as the path to the tree will still be '['
+ * since we removed encoding logic from the driver layer with this PR: https://github.com/microsoft/FluidFramework/pull/21680
+ *
+ * ADO:18003
+ */
+describeCompat.skip(
```

Comment:
Consider this alternative to skipping it - Validate that the summarize fails instead of passing.
That way you have the test running and when the bug is fixed, this test will start failing and will require to be fixed (and not missed).

Something like this - https://github.com/microsoft/FluidFramework/blob/36e8ec469b7a8d2b5cceec3c21dee27fe7913563/packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts#L962

Reply:
@agarwal-navin  / @markfields 
I am choosing to keep it skipped. The test runs just too long + I wish to keep the rest of code after the failure is encountered for completeness which. I have made sure to mention this test in the ADO item to ensure it is not missed.

Reply:
Sure, I will leave it up to you. IMO, test running too long should not be a concern because thats not different than other tests.
Also, mentioning in ADO doesn't seem to be enough in my experience, especially when it has been sitting for a while and some one else takes up the work :-).